apps/music/components/gaia-fast-list/gaia-fast-list.js

```diff
@@ -356,14 +382,24 @@ var GaiaFastListProto = {
         text-overflow: ellipsis;
       }
 
-      ::content .image ~ h3,
-      ::content .image ~ p {
-        padding-inline-end: 60px;
+      ::content :-moz-dir(ltr) .image ~ h3,
+      ::content :-moz-dir(ltr) .image ~ p {
+        padding-right: 52px;
+      }
+
+      ::content :-moz-dir(rtl) .image ~ h3,
+      ::content :-moz-dir(rtl) .image ~ p {
+        padding-left: 52px;
```

Comment:
I would double-check that this looks right in RTL for the search results list. I tried doing this in a separate patch, and found that -moz-dir(rtl) doesn't actually work (even with the directionality hacks that gaia-components use) in a web component used by another web component.